### ISSUE TITLE
Built-In Self-Update

### ISSUE THREAD

**Issue by stamoun** [dvx/lofi#72]
The ability to update the application when a new release is published. For example, Discord does this as well as BrewFather (https://web.brewfather.app yes I brew my own beer 🍺).

I'm wondering if this could be done through CI/CD (AppVeyor for instance, which is free for OSS project IIRC). This may be overkill for this project, but could be something fun to work on :)

p.s.: This is definitely an enhancement, I don't know how to tag these (I probably don't have the right to)

**Comment by stamoun** [comment]
Well well, this didn't take much research :) https://www.electronjs.org/docs/tutorial/updates#using-updateelectronjsorg

That being said, I'm wondering if a CI/CD could be useful, that would be a new issue (I used to be a build master, so I tend to really like CI/CD).

**Comment by dvx** [comment]
Yeah, this is something I've been looking to implement. It's actually not too difficult, but I do think there needs to be a REST endpoint set up somewhere.
